src/bench/xor.cpp

```diff
@@ -3,22 +3,1283 @@
 // file COPYING or https://opensource.org/license/mit/.
 
 #include <bench/bench.h>
+#include <cmath>
+#include <cstddef>
+#include <map>
```

Comment:
You can keep the std lib headers separated from our own headers.

Reply:
[Done](https://github.com/bitcoin/bitcoin/compare/8696184c2fd183483fa843348323ffbc62c2de3c..33625e9d5327263c273583d7b3fe2f3114ebf12e), anything else?